internal/provider/providers/porkbun/api.go

```diff
@@ -8,19 +8,30 @@ import (
 	"net/http"
 	"net/url"
 	"strconv"
+	"strings"
 
 	"github.com/qdm12/ddns-updater/internal/provider/errors"
 )
 
+type dnsRecord struct {
+	ID       string `json:"id"`
+	Name     string `json:"name"`
+	Type     string `json:"type"`
+	Content  string `json:"content"`
+	TTL      string `json:"ttl"`
+	Priority string `json:"prio"`
```

Comment:
Could comment out the unused fields here, which would make the API less "brittle" in the event that porkbun does change what they're returning. I don't really expect them to do that, but might be worth doing since priority / notes are unused, though this is their current api model.